Yoast/Yoast-theme-public
# Stories/1224 move login and cart to header

Fixes https://github.com/Yoast/yoast.com/issues/1224

Goes along with: https://github.com/Yoast/yoast.com-ms/pull/108

**how to test**
Check if there is a new top navigation structure that has Login, Cart and Search. When focussing the search field, it slides out. When the input has no value, it will slide back if not focused. If it has a value, it stays "wide".

Important to check if it works all around the Yoast site, as the header is used on kb. academy, and the default site.
